src/core/js/views/blockView.js

```diff
@@ -1,16 +1,22 @@
 define([
     'core/js/views/adaptView'
-    ], function(AdaptView) {
+], function(AdaptView) {
 
     var BlockView = AdaptView.extend({
 
+        attributes: function() {
```

Comment:
@oliverfoster i didn't change adaptView so it wouldn't inherit this from there. or are you saying you think i 'should' put this in adaptView and not in its subclasses?